packages/framework-integration-tests/integration/end-to-end/setup.ts

```diff
@@ -0,0 +1,43 @@
+import util = require('util')
+const exec = util.promisify(require('child_process').exec)
+import { deploy, nuke } from '../../src/deploy'
+import { config } from 'aws-sdk'
+
+// TODO: remove the AWS config with BOOST-729 to make end-to-end tests provider unaware
```

Comment:
👍

Reply:
Can end-to-end tests actually be provider-unaware? Passing these tests in AWS doesn't mean that they'll pass in Azure...

Reply:
Oh, nono, of course, we need to pass these tests using each provider, but the tests themselves are the same (I mean the same file, the same code), we don't need to repeat them. That's why we want these tests to only use the public API of Booster.

This way, we have the following benefits:
1.- We get tests for free for all the providers
2.- The most important: we will ensure all providers behave the same way regardless the provider used 💥